include/vcpkg/base/parallel-algorithms.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#if defined(USE_PARALLEL_ALG) && !defined(GNU_USE_PARALLEL_ALG)
+#include <algorithm>
+#include <execution>
+#elif defined(GNU_USE_PARALLEL_ALG)
+#include <parallel/algorithm>
+#else
+#include <algorithm>
+#endif
+
+
+#if defined(USE_PARALLEL_ALG) && !defined(GNU_USE_PARALLEL_ALG)
+#define vcpkg_parallel_for_each std::for_each(std::execution::par,
```

Comment:
This is significantly faster than using a real function because it doesn't get inlined.